serverless/template.yml

```diff
@@ -82,60 +94,106 @@ Resources:
                   - "logs:PutLogEvents"
                 Resource:
                   Fn::Sub: "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/PrometheusTimestreamConnector-LambdaFunction-*:*"
+              - Effect: "Allow"
+                Action:
+                  - "timestream:DescribeEndpoints"
+                Resource: "*"
+              - Effect: "Allow"
+                Action:
+                  - "timestream:WriteRecords"
```

Comment:
If the connector is deployed with SigV4 authentication enabled, it will ignore basic auth headers and try to use local credentials. This means that the Lambda function needs to have all necessary execution permissions.